src/math/max.js

```diff
@@ -0,0 +1,18 @@
+const max = (numArray) => {
+let currentHighNumber = 0;
+  if(numArray[0] > numArray[1]) {
+    currentHighNumber =numArray[0];
+  } else if (numArray[1] > numArray[0]) {
+    currentHighNumber = numArray[1];
+    }
+
+  for(let i = 2; i < numArray.length; i++) {
+    if(numArray[i] > currentHighNumber) {
+      currentHighNumber = numArray[i];
+    } else {
```

Comment:
This else is also redundant. 
You are either assigning `currentHighNumber` to a new value or reassigning it to itself, which you don't need to do, currentHighNumber doesn't get reset back to 0 if you don't reassign it to itself in the else clause of your for loop.
You can delete lines 12 - 14